Updated function that reads PSON settings to support UTF-8 characters.

lucasbode opened the pull request:
Addressing issue #19

JasonRowe commented:
Maybe we should update all the uses of Get-Content?

https://github.com/powerdeploy/Powerdeploy/search?p=2&q=Get-Content&utf8=%E2%9C%93

jasonmueller commented:
I believe this change is to enable proper handling of Japanese characters?  Could you write some tests to verify that behavior and then prove it works with this code?  Powerdeploy uses Pester for testing.

lucasbode commented:
Yes, there were some Japanese characters that were breaking the process. I went ahead and updated the Get-Content occurrences to use UTF-8. 

I also went ahead and added tests to Install-Deployment.Tests.ps1 and GetFilesystemConfiguration.tests.ps1 for some UTF-8 setttings. 

I ran the build command and the tests passed as well as the acceptance tests. Let me know if there's anything else you'd like to see added.
